docroot/themes/humsci/humsci_basic/src/scss/components/_main-nav.scss

```diff
@@ -26,5 +16,89 @@
       background-image: svg('<svg viewBox="0 0 24 24"><path d="M10.56,12,.32,22.21a1,1,0,0,0,1.41,1.41L12,13.39,22.21,23.62a1,1,0,0,0,1.41-1.41L13.39,12,23.62,1.74a1,1,0,0,0,0-1.42,1,1,0,0,0-1.41,0L12,10.56,1.74.32A1,1,0,0,0,.32,1.74Z" fill="#2e2d29" transform="translate(-0.03 -0.03)"/></svg>') !important;
     }
   }
-  // scss-lint:enable ImportantRule
+
+  &__menu {
+    box-shadow: none !important;
+    transform: translate(0) !important;
+    width: auto !important;
+
+    list-style-type: none;
+    margin: 0;
+    padding: 0;
+
+    line-height: 2;
+
+    @include grid-media-min('sm') {
+      max-width: none;
+    }
+
+    &-lv1 {
+      position: relative !important;
+
+      &[aria-hidden="true"] {
+        display: none !important;
+      }
+
+      @include grid-media('md') {
+        display: flex;
+        max-width: none !important;
+      }
+
+      @include grid-media('lg') {
+        .su-main-nav__menu-container {
+          position: absolute;
+        }
+      }
+    }
+
+    &-lv2 {
+      @include grid-media('lg') {
+        // Hide any nested menus that are level 3 or greater
+        .su-main-nav__button,
+        .su-main-nav__menu-container {
+          display: none;
+        }
+      }
+    }
+  }
+
+  &__menu-container {
+    padding-left: 20px;
+    margin-top: 8px;
+    margin-bottom: 8px;
+
+    &[aria-hidden="true"] {
```

Comment:
do we need some noJS fallbacks for aria-hidden classes that may or may not happen if JS is turned off?

Reply:
you can use the .no-js class that will be on the top html element too